app/task/Install.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/*
+  +------------------------------------------------------------------------+
+  | Phosphorum                                                             |
+  +------------------------------------------------------------------------+
+  | Copyright (c) 2013-present Phalcon Team (https://www.phalconphp.com)   |
+  +------------------------------------------------------------------------+
+  | This source file is subject to the New BSD License that is bundled     |
+  | with this package in the file LICENSE.txt.                             |
+  |                                                                        |
+  | If you did not receive a copy of the license and are unable to         |
+  | obtain it through the world-wide-web, please send an email             |
+  | to license@phalconphp.com so we can send you a copy immediately.       |
+  +------------------------------------------------------------------------+
+*/
+
+namespace Phosphorum\Task;
+
+use Phalcon\Di;
+use League\CLImate\CLImate;
+use Phosphorum\Console\AbstractTask;
+use Phosphorum\Console\OptionParser;
+use Phosphorum\Exception\InvalidParameterException;
+
+/**
+ * Phosphorum\Task\Install
+ *
+ * @package Phosphorum\Task
+ */
+class Install extends AbstractTask
+{
+    protected $pathesList = [
+        'assets_path',
+        'log_path',
+        'pids_path',
+        'annotations_cache',
+        'config_cache',
+        'data_cache',
+        'metaData_cache',
+        'views_cache',
+        'volt_cache',
+    ];
+
+    protected $user = [
+        'userName' => 'www-data',
+    ];
+
+    /**
+     * @Doc("Check folders that must have permission to write files into them")
+     */
+    public function checkDirs()
```

Comment:
@sergeysviridenko Could you please rename this to `check` to have command like `install:check` not  `install:checkDirs`